core/Updates/4.5.0-b1.php

```diff
@@ -29,11 +29,16 @@ public function __construct(MigrationFactory $factory)
 
     public function getMigrations(Updater $updater)
     {
-        $migration1 = $this->migration->db->changeColumnType('session', 'data', 'MEDIUMTEXT');
+        $migrations = [];
+        $migrations[] = $this->migration->db->changeColumnType('session', 'data', 'MEDIUMTEXT');
+
+        //database using utf8mb4 set row format dynamic
+        if (DbHelper::getDefaultCharset() == "utf8mb4") {
```

Comment:
@peterhashair because we have already release 4.5.0-b1 we would usually need to put this migration into a new file `4.5.0-b2`. It's not too important maybe for this one so we can leave it for now and don't need to change it.

Reply:
@tsteur Removed.

Reply:
@peterhashair maybe you forgot to commit the new update fail? Not seeing the migration anymore anywhere